### PR TITLE
Guidance restructure

### PR DESCRIPTION
**Pull request by liamwilson-kainos**
Includes the following tickets -

* 193147
* 193146
* 193144
* 193149
* 193503

### CODE REVIEW

**Review by marcin-mogiela — DISMISSED** [review]

File: app/views/guidance/landing-page.njk [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      </div>
+      <p class="govuk-body govuk-!-font-size-24">
+      Farmers who keep cattle, sheep and pigs can get funding to pay for a vet to visit their farm and carry out an annual health and welfare review.
+      </p>
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      <p class="govuk-body-s">From: <a class="govuk-link govuk-!-font-weight-bold" href="https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs">Department for Environment, Food &amp; Rural Affairs</a> and <a class="govuk-link govuk-!-font-weight-bold" href="https://www.gov.uk/government/organisations/rural-payments-agency">Rural Payments Agency</a></p>
+    </div>
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      A health and welfare review of livestock provides funding for a vet, or team chosen by a vet, to visit a farm and carry out a health and welfare review of beef or dairy cattle, sheep or pigs.
```
[the remark this answers is not in the document]
Comment (liamwilson-kainos) [reply app/views/guidance/landing-page.njk:20]: good spot ✅

File: app/views/guidance/landing-page.njk [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    <ul class="govuk-list govuk-list--bullet">
+      <li>reduce endemic diseases and conditions</li>
+      <li>increase animal productivity</li>
+      <li>improve animal welfare</li>
+    </ul>
+    <p class="govuk-body">
+      The review is optional. It is not an inspection and will not be used for compliance or monitoring.
+    </p>
+    <h2 class="govuk-heading-m">Farmers</h2>
+    <p class="govuk-body">Read the apply guidance to find out:</p>
```
Comment (chrissmKainos) [inline app/views/guidance/landing-page.njk:31]: According to the word document, this should read 

`Read the apply guidance to check:`

File: app/views/guidance/landing-page.njk [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    <ul class="govuk-list govuk-list--bullet">
+      <li>if you’re eligible</li>
+      <li>how much funding you can get</li>
+     <li>what you must ask the vet to do when they’re visiting your farm</li>
+    </ul>
+    <p class="govuk-body">Read the claim guidance to find out:</p>
```
Comment (chrissmKainos) [inline app/views/guidance/landing-page.njk:37]: According to the word document, this should read

`Read the claim guidance to find out:`

File: app/views/guidance/landing-page.njk [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            <time datetime="2016-06-27T10:29:44+00:00">10 May 2023</time>
+          </li>
+          <li class="gem-c-document-list__attribute">Guidance</li>     
+        </ul>
+      </li>
+      <li class="gem-c-document-list__item">
+        <a class="gem-c-document-list__item-title govuk-link" href="apply/claim-guidance-for-farmers">How to claim for an annual health and welfare review of livestock</a>
+        <ul class="gem-c-document-list__item-metadata">
+          <li class="gem-c-document-list__attribute">
+            <time datetime="2015-09-24T16:42:48+00:00">17 May 2023</time>
```
Comment (chrissmKainos) [inline app/views/guidance/landing-page.njk:57]: According to the word document, this should read

`17 April 2023`

File: app/views/guidance/landing-page.njk [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      <li>what information you’ll need to share when you claim</li>
+      <li>what evidence you’ll need to keep</li>
+    </ul>
+
+    <ul class="gem-c-document-list govuk-!-margin-bottom-6">
+      <li class="gem-c-document-list__item">
+        <a class="gem-c-document-list__item-title   govuk-link" href="apply/guidance-for-farmers">How to apply for an annual health and welfare review</a>
+        <ul class="gem-c-document-list__item-metadata">
+          <li class="gem-c-document-list__attribute">
+            <time datetime="2016-06-27T10:29:44+00:00">10 May 2023</time>
```
Comment (chrissmKainos) [inline app/views/guidance/landing-page.njk:48]: According to the word document, this should read

`17 April 2023`

File: app/views/guidance/claim-guidance-for-farmers.njk [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your letter or email.</p>
+
+      <div class="govuk-inset-text">
+        <p class="govuk-body">Rural Payments Agency<br>
+        PO Box 352<br>
+        Worksop<br>
+        S80 9FG<br>
+        </p>
+
+        {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
```
Comment (chrissmKainos) [inline app/views/guidance/claim-guidance-for-farmers.njk:96]: There is an additional blank line prior to the "Find out about call charges" link within the build that isn't in the prototype

**Review by json-hamilton — APPROVED** [review]
Ran locally and looks good to me once the content is fixed👌

File: app/views/guidance/apply-guidance-for-farmers.njk [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your letter or email.</p>
+
+      <div class="govuk-inset-text">
+        <p class="govuk-body">Rural Payments Agency<br>
+        PO Box 352<br>
+        Worksop<br>
+        S80 9FG<br>
+        </p>
+
+        {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
```
Comment (chrissmKainos) [inline app/views/guidance/apply-guidance-for-farmers.njk:311]: There is an additional blank line prior to the "Find out about call charges" link within the build that isn't in the prototype

File: app/views/guidance/guidance-for-vet.njk [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+        <li>increase animal productivity</li>
+        <li>improve animal welfare</li>
+      </ul>
+
+      <p class="govuk-body">You may be asked by a farmer to carry out a review. It is not an inspection or audit and is optional.</p>
+
+      <h3 id="what-a-farmer-will-ask-you-to-do" class="govuk-heading-s">What a farmer will ask you to do</h3>
+
+      <ul class="govuk-list govuk-list--number">
+        <li>Visit the farm and test for the endemic diseases or condition of the livestock type you’re reviewing to recommended standards - bovine viral diarrhoea (BVD) in dairy and beef cattle, effectiveness of worming treatments in sheep, porcine reproductive and respiratory syndrome (PRRS) in pigs.</li>
```
Comment (chrissmKainos) [inline app/views/guidance/guidance-for-vet.njk:73]: The style of the numbers within the list do not match the prototype which are white numbers in black circles

File: app/views/guidance/apply-guidance-for-farmers.njk [hunk 9]
```diff
@@ -0,0 +1,319 @@
...
+      <p class="govuk-body">The funding will cover the endemic disease or condition testing costs. It will also contribute towards the costs of both your time and the vet’s time.</p>
+
+      <h2 id="what-happens-during-a-review" class="govuk-heading-m">What happens during a review</h2>
+
+      <p class="govuk-body">There’s no set amount of time for a review, but it’s likely to take between 2 and 3 hours of your time and the vet’s time. You may ask the vet to spread it over several regular visits to the farm or do it in a single visit.</p>
+
+      <h3 id="what-the-vet-must-do" class="govuk-heading-s">What you must ask the vet to do</h3>
+
+      <ul class="govuk-list govuk-list--number">
+        <li>Visit your farm and test for the endemic diseases or condition of the livestock type you’re having reviewed to recommended standards - bovine viral diarrhoea (BVD) in dairy and beef cattle, effectiveness of worming treatments in sheep, porcine reproductive and respiratory syndrome (PRRS) in pigs.</li>
```
Comment (chrissmKainos) [inline app/views/guidance/apply-guidance-for-farmers.njk:114]: The style of the numbers within the list do not match the prototype which are white numbers in black circles

File: app/views/guidance/guidance-for-vet.njk [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+        <p class="govuk-body">Reviewing biosecurity and medicine usage are optional parts of the review.</p>
+      </div>
+      
+      <h3 id="review-biosecurity" class="govuk-heading-s">Review biosecurity (optional)</h3>
+
+      <p class="govuk-body">The farmer may ask you to review biosecurity measures, explain the benefits of good biosecurity and give recommendations for improving it.</p>
+
+      <p class="govuk-body">For external biosecurity, this could include:</p>
+
+      <ul class="govuk-list govuk-list--number">
```
Comment (chrissmKainos) [inline app/views/guidance/guidance-for-vet.njk:156]: this is a bulleted list in the prototype

File: app/views/guidance/guidance-for-vet.njk [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+        <li>farm visitors and vehicles</li>
+        <li>wildlife</li>
+        <li>interaction with other farm animals</li>
+        <li>quarantine plans</li>
+      </ul>
+
+
+      <p class="govuk-body">For internal biosecurity, this could include:</p>
+
+      <ul class="govuk-list govuk-list--number">
```
Comment (chrissmKainos) [inline app/views/guidance/guidance-for-vet.njk:167]: this is a bulleted list in the prototype

File: app/views/guidance/guidance-for-vet.njk (hunk 8 above)
Comment (liamwilson-kainos) [reply app/views/guidance/guidance-for-vet.njk:73]: Thanks - this one has been accepted as not possible with our component toolkit

File: app/views/confirmation.njk [hunk 12]
```diff
@@ -43,7 +45,7 @@
       <a class='govuk-link' href={{ applySurveyUri }}>
         What did you think of this service?
       </a>
-      (takes 30 seconds)
+      (giving feedback takes about 5 minutes)
```
[the remark this answers is not in the document]
Comment (liamwilson-kainos) [reply app/views/confirmation.njk:48]: added it and aligned to the button used elsewhere

File: app/views/guidance/guidance-for-vet.njk [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+        <li>your name and Royal College of Veterinary Surgeons (<abbr title="Royal College of Veterinary Surgeons">RCVS</abbr>) number</li>
+        <li>confirmation you have given the farmer a written report</li>
+      </ul>
+  
+    <div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
+      <button onclick="window.print()" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
+    </div>
+    </div>
+  </div>    
+{% endblock %}
```
[the remark this answers is not in the document]
Comment (liamwilson-kainos) [reply app/views/guidance/guidance-for-vet.njk:240]: Added - checking other pages as well :)

**Review by chrissmKainos — APPROVED** [review]

**Pull request merged**